src/calls/components/SingleCallsView.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class SingleCallsView extends Component {
+  render() {
+    return (
+      <div>
+        <h1>Pojedinacan poziv</h1>
```

Comment:
We're squarely in UTF-8 land with our source files, so feel free to use 'č' / 'ć', etc. 🙂

Reply:
Typo is fixed.